test/lib/auth.js

```diff
@@ -63,6 +69,179 @@ describe('Auth lib', function () {
         };
     };
 
+    describe('Update party API', function () {
+	
+	var auth = getAuthLib(strategy, {});
+	var req = {
+	    id: 'test_user',
+	    appId: config.oauth2.clientID,
+	    user: {accessToken: 'token',
+		   id: 'rick',
+		   nickName: 'theMagician',
+		   organizations: [{
+		       id: '123456789',
+		       name: 'patty',
+		       roles: [{
+			   id: '3',
+			   name: 'Seller'
+		       }, {
+			   id: '6',
+			   name: 'provider'
+		       }]
+		   }, {
+		       id: '987654321',
+		       name: 'MntyPythn',
+		       roles: [{
+			   id: '6',
+			   name: 'provider'
+		       }]
+		   }, {
+		       id: '111555999',
+		       name: 'AmaneceQueNoEsPoco',
+		       roles: [{
+			   id: '3',
+			   name: 'Seller'
+			}, {
+			    id: '1',
+			    name: 'purchuaser'
+			}]
+		   }]
+		  }
+        };
+	
+	it ('should continue with middleware processing if not user object has been provided', function (done) {
+	    var req = {
+		id: 'test_user',
+		appId: config.oauth2.clientID,
+		accesstoken: 'token'
+            };
+	    auth.checkOrganizations(req, {}, function () {
```

Comment:
If you are not going to make any validation you can include directly "done" as callback